### PR TITLE
365 assign docker container to camera

### LINKED ISSUE #365: Assign camera to docker via serial number
So far, the cameraconnect module will randomly pick a camera which is available in the network. If more than one camera is available the user has no option to specify which one should be used. 

Solution: The user should be able to specify the camera via its serial number within the env file of the container

### PR DESCRIPTION
**Pull request by JonaVo**
Fixes #365 
Multi camera support now available
User specifies camera via mac address, which is written on the back of each camera

Also adjusted the harvester version used

### CODE REVIEW

**Comment by JonaVo** [comment]
Please check again, except the 16 parameters for the init function, smelly code has been removed

File: cameraconnect/src/cameras.py [hunk 1]
```diff
@@ -137,7 +140,7 @@ def _publish_mqtt(self, image) -> None:
         message = json.dumps(prepared_message)
         # Publish the message
         ret = self.client.publish(self.mqtt_topic, message, qos=0)
-        print(ret)
+        print("Image No.: " + str(ret[1]))
```
Comment (JeremyTheocharis) [inline cameraconnect/src/cameras.py:143]: Please use logging module for each use of print

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: cameraconnect/src/cameras.py (hunk 1 above)
Comment (JeremyTheocharis) [reply cameraconnect/src/cameras.py:139]: @JonaVo Please take a look at the logging (using the logging instead of print)

Comment (JonaVo) [reply cameraconnect/src/cameras.py]: done 1bc5eed

**Pull request merged**